src/main/scala/cromwell/util/docker/DockerRegistry.scala

```diff
@@ -0,0 +1,23 @@
+package cromwell.util.docker
+
+/**
+  * A set of endpoints belonging to a registry.
+  */
+object DockerRegistry {
+  /**
+    * For everything _but_ Docker Hub, the namespace == registry v1 host == registry v2 host.
```

Comment:
Something seems off about this comment... having trouble parsing `namespace == registry v1 host == registry v2 host.`